### ISSUE TITLE
Braces not added around the last case/default block of a switch statement

### ISSUE THREAD

**Issue by micheleCTDE** [uncrustify/uncrustify#3045]
uncrustify 0.72.0 + backport of commits 45681e88, 64d93e3.
Files in attachment:
- t1_in.cpp: test file (subset of a real file)
- t1_out.cpp: output of the test run
- test.cfg: configuration used for the test

Step to reproduce: run "uncrustify -c test.cfg -f t1_in.cpp -o t1_out.cpp"

Braces are not added to the last block of a switch, regardless of whether it is a "default" or a "case" block.

Current output:
```
void main()
{
	switch (af)
	{
		case AF_INET:
		{
			return 1;
		}

		case AF_INET6:
		{
			return 1;
		}

		default:
			return 0;
	}
}
```

Note how braces are correcly added for the first 2 case blocks but not the the last one.

[missing_last_braces.zip](https://github.com/uncrustify/uncrustify/files/5763926/missing_last_braces.zip)

**Comment by micheleCTDE** [comment]
This is fixed by PR #3048. 
Once #3048 is fixed up, I will verify again and then we can probably close this report.

**Comment by micheleCTDE** [comment]
This is fixed by PR #3056.

**Comment by micheleCTDE** [comment]
PR #3056 has been merged, so closing this issue.
